controller/client/client.go

```diff
@@ -486,9 +486,9 @@ outer:
 
 // StreamJobEvents streams job events to the output channel.
 func (c *Client) StreamJobEvents(appID string, output chan *ct.JobEvent) (stream.Stream, error) {
-	appEvents := make(chan *ct.AppEvent)
-	go convertAppEvents(appEvents, output)
-	return c.ResumingStream("GET", fmt.Sprintf("/apps/%s/events?object_type=%s", appID, ct.EventTypeJob), appEvents)
+	appEvents := make(chan *ct.Event)
+	go convertEvents(appEvents, output)
+	return c.ResumingStream("GET", fmt.Sprintf("/apps/%s/events?object_types=%s", appID, ct.EventTypeJob), appEvents)
```

Comment:
We should probably just move to a single path for events and just return `StreamEvents({AppID: appID, ObjectTypes: [ct.EventTypeJob]}` here